canonical-web-and-design/react-components
# Set the button type of the contextual menu so it doesn't submit forms

## Done

- Set the contextual menu button type to "button"

## QA
- Run `dotrun link-package` in react components.
- Run `dotrun link-components` in maas-ui.
- Open the KVM compose form (on a KVM details page).
- Under `Storage configuration` click on the select box in the Location column.
- You should not see a warning in the console about submitting the form.
